docs/pages/contributing/documentation/terms-usage.mdx

```diff
@@ -0,0 +1,171 @@
+## Terminology and usage conventions
+
+This section covers common terminology, style, and usage questions.
+It includes recommended practices for using punctuation, referring to user interface
+elements, and selecting the right words for the information you're trying to convey. 
+
+### Above and below
+
+Don't use *above* to mean *earlier* or as an adjective preceding a noun (*the above section*) 
+or following a noun (*the code above*). Use a hyperlink, or use *previous*, *preceding*, or 
+*earlier*. Don't use *below* to mean *later* or as an adjective preceding a noun (*the below 
+section*) or following a noun (*the code below*). Use a hyperlink, or use *later* or 
+*the following*. For example: 
+
+- Use the preceding code to display information about the database.
+- Use the following code to display information about the database.
+
+### Dates and numbers
+
+Date formats can depend on context, including the location of a computer and a user's 
+preferences. In most cases, you can use the DD Mon YYYY or DD Month YYYY format for dates. 
+In body text, spell out whole numbers from zero through nine. Use numerals for 10 or greater. 
+Use commas in numbers with four or more digits. Use *more than* instead of *over* 
+(over is a spatial term). 
```

Comment:
> Use *more than* instead of *over* (over is a spatial term). 

This seems artificially limiting to me. I see "over" all the time as an expression of quantity. (e.g., if you google `"with over" site:nytimes.com`).

Reply:
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/o/over